### DISCUSSION TITLE
Is a release 0.11 planned anytime soon or should I run from main ?

### DISCUSSION THREAD

**Discussion by primeroz** [prometheus-operator/kube-prometheus#1785] [General] [answered]
Hi,

i need to run an updated vrersion of the operator compared to what is in 0.10 , there are changes to the CRD and RBAC though so i would have to do more than just update the version and i'd rather not mess up my codebase too much

Is a release planned anytime soon ? 

thanks

**Comment by paulfantom** [comment, marked as answer]
Yes, release is planned to happen in upcoming days. There is a WIP PR already in https://github.com/prometheus-operator/kube-prometheus/pull/1784

**Reply by primeroz** [reply]
thank you, i must have searched badly :+1:

maybe we could label release PR with a `release` label so they are easy to find ?

**Reply by paulfantom** [reply]
It is the first PR on the list and has `release` in title. I don't see any reason to create additional labels just for releases :)

**Reply by primeroz** [reply]
fair enough, i realized my mistake now. i looked in issues not pr 

thanks a lot for the amazing work
